src/app/settings/organization-plans.component.html

```diff
@@ -165,6 +165,13 @@ <h2 class="spaced-header">{{'summary' | i18n}}</h2>
                         {{selectablePlan.basePrice | currency:'$'}}
                         /{{'year' | i18n}}
                     </small>
+                    <small *ngIf="acceptingSponsorship">
+                        {{'basePrice' | i18n}}: {{ 40 / 12 | currency:'$'}} $times; 12
```

Comment:
This $40 should be the `basePrice` so that we can update the price as we want

Reply:
The screenshot shows `&times;` rather than `x`, not sure why.

Reply:
I change `basePrice` to 0 so that the rest of the calculations will show the correct number. I don't love hardcoding it either though. But to not hardcode it we would need some sort of `originalBasePrice` on the `PlanResponse`.

Reply:
No longer doing this.